react-native/react/pinentry/index.js

```diff
@@ -2,7 +2,7 @@ import React, {Component} from '../base-react'
 import {connect} from '../base-redux'
 import PinentryRender from './index.render'
 
-export class Pinentry extends Component {
+class Pinentry extends Component {
```

Comment:
The connect call is exported. That's all that's needed right

On Thursday, December 10, 2015, Marco Munizaga <notifications@github.com>
wrote:

> In react-native/react/pinentry/index.js
> <https://github.com/keybase/client/pull/1507#discussion_r47304360>:
>
> > @@ -2,7 +2,7 @@ import React, {Component} from '../base-react'
> >  import {connect} from '../base-redux'
> >  import PinentryRender from './index.render'
> >
> > -export class Pinentry extends Component {
> > +class Pinentry extends Component {
>
> Why isn't this exported?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1507/files#r47304360>.
>